cmd/buildah/bud.go

```diff
@@ -358,6 +358,7 @@ func budCmd(c *cobra.Command, inputArgs []string, iopts budOptions) error {
 		OciDecryptConfig:        decConfig,
 		Jobs:                    &iopts.Jobs,
 		LogRusage:               iopts.LogRusage,
+		RusageLogFile:           iopts.RusageLogFile,
```

Comment:
I know the three prior lines aren't in alpha order, but this one and the others ought to be

Reply:
I've put it in the right place (I hope).